eng/common/tools.ps1

```diff
@@ -378,7 +378,16 @@ function InitializeVisualStudioMSBuild([bool]$install, [object]$vsRequirements =
   }
 
   $msbuildVersionDir = if ([int]$vsMajorVersion -lt 16) { "$vsMajorVersion.0" } else { "Current" }
-  return $global:_MSBuildExe = Join-Path $vsInstallDir "MSBuild\$msbuildVersionDir\Bin\msbuild.exe"
+
+  $local:BinFolder = Join-Path $vsInstallDir "MSBuild\$msbuildVersionDir\Bin"
+  $local:Prefer64bit = if ($vsRequirements.Prefer64bit) { $vsRequirements.Prefer64bit } else { $false }
```

Comment:
Found the problem that's on this line. All goes back to `set-strictmode -version 2.0` which exists in the release/5.0 and main copies of eng/common/tools.ps1 but not the release/3.x one. See https://github.com/dotnet/wpf/pull/4750/files#diff-72b8f8e899b94872c6ead31fd06ec109da15bcb9ad2af6e78103d6763a31c637R59

Quick fix

```suggestion
  $local:Prefer64bit = if (Get-Member -InputObject $vsRequirements -Name 'Prefer64bit') { $vsRequirements.Prefer64bit } else { $false }
```

Tested this locally in dotnet/aspnetcore with same Arcade version

Reply:
If this is urgent, take my suggestion even though this is an eng/common/ file. Wait for another Arcade build and validation if not.

Reply:
Fix for 'main' coming in dotnet/arcade#7564 @mmitche has already approved it (you are now pre-approved 😄)